app/views/applications/show.html.haml

```diff
@@ -14,7 +14,7 @@
   #application
     %article#application-info
       %header
-        %h3.address= @application.address
+        %h1.address= @application.address
```

Comment:
I have changed it back to the original one and will submit a separate PR when this PR gets merged.